### ISSUE TITLE
Feature request: add support for go:generate comments

### ISSUE THREAD

**Issue by dmitry-vovk** [go-lang-plugin-org/go-lang-idea-plugin#1460]
As for Go 1.4, there is ```go generate support``` in line comments. Example:
```go
//go:generate go tool yacc -o gopher.go -p parser gopher.y
```
As in [Generating code](http://blog.golang.org/generate) blog article.

Please add support for this feature:
* syntax highlighting
* command to run go generators

**Comment by dlsniper** [comment]
Hi, what exactly do you expect from the syntax highlighting part? Aside from ```//go:generate```, there's nothing else that's standard about it.

I could see how running ```go generate``` before the build / run step could be useful.

**Comment by dmitry-vovk** [comment]
It would be nice to have go:generate colored differently, to easily spot among plain line comments.

**Comment by dlsniper** [comment]
Btw, running ```go generate``` is possible right now. You need to edit your run configuration and add a new ```Go command``` to the ```Before launch section```.

That should do the trick (I haven't tested it as I don't use this part of the plugin at all)

![snapshot21](https://cloud.githubusercontent.com/assets/607868/6740415/e28f7b38-ce7f-11e4-92ff-ba5bc4e3a9fb.png)

**Comment by dmitry-vovk** [comment]
Yes, I use that. But it would be nice to have "Go generate" command beside "Go fmt" and "goimports" commands in Code menu. Sometimes I want to inspect generated files, and switching between IDE and console is not very productive.

**Comment by dlsniper** [comment]
If you want that, then I can suggest to try and implement something like the menu option for ```go fmt``` for the whole project.
It should be straightforward if you use https://github.com/go-lang-plugin-org/go-lang-idea-plugin/blob/v1.0.0-alpha0/src/com/goide/actions/fmt/GoFmtProjectAction.java as a blueprint for it.
